### DISCUSSION TITLE
Daily 25/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#157] [Daily Sprint]
- Que hice ayer?
- Que haré hoy?
- Dificultades o Impedimentos.

**Comment by NataliaAlvarezIspc** [comment]
Ayer no pude hacer nada con respecto al proyecto.
Hoy estaré empezando a trabajar en las issue que me asigné y terminando de ver videos sobre Angular. Por la noche tengo clase de Inglés.

**Comment by rpgrca** [comment]
Hoy estuve en la clase de programación web, y empecé a jugar con Angular, armé el formulario de contacto con los estilos del sitio Bidón, voy a seguir con el encabezado y el pie de página.

**Comment by Brian-Almada** [comment]
Ayer estuve viendo algo de Angular. Hoy asistí a la clase de Prog.Web e Inglés. Ahora sigo con un curso de Angular para entender conceptos

**Comment by BraianTroncoso** [comment]
Ayer estuve en una clase de Python y Js sobre OOP. Hoy estuve con clases de Ingles, y hoy voy a ver un poco el código del sitio en Angular. Voy a leer más sobre angular y ver algún proyecto, estoy con un curso pero lleva tiempo.
